README.md

```diff
@@ -1,20 +1,20 @@
-# **SK**etch **E**fficiently a **WE**b of **R**elationships
-SKEWER allows a coarse grained structure to be visualized in pymol like an atomistic structure by drawing bonds and elastic network.
+# Garnish Accurately Represents a Network-Interconnected System in a Hearbeat
```

Comment:
Hey awesome you can add comments to lines!
Anyway you missed a t in heartbeat :P
```suggestion
# Garnish Accurately Represents a Network-Interconnected System in a Heartbeat
```